Packages/ClientRuntime/Sources/Networking/Http/Headers.swift

```diff
@@ -161,6 +161,12 @@ public struct Headers: Equatable, Hashable {
     }
 }
 
+extension Headers: Equatable {
+    public static func == (lhs: Headers, rhs: Headers) -> Bool {
```

Comment:
This and the other public operator overloads should have doc comments to explain their behavior (i.e. insensitive to the order of the headers array)

Reply:
for sure. fixed.